netdcli/netdcli.go

```diff
@@ -295,12 +295,13 @@ func validateOpts(opts *cliOpts) error {
 	if opts.pktTag == "auto" {
 		if opts.oper.Get() == CLI_OPER_CREATE &&
 			opts.construct.Get() == CLI_CONSTRUCT_NW {
-			log.Printf("  auto allocating network subnet from global pool")
+			log.Infof("  auto allocating network subnet from global pool")
 		}
 	} else if opts.pktTag != "" {
 		_, err = strconv.Atoi(opts.pktTag)
 		if err != nil {
 			log.Fatalf("Error convertinng tag %s to integer \n", opts.pktTag)
+			os.Exit(1)
```

Comment:
It does in `log` but not in logrus. This is one of the ways it differs.

I will look at encapsulating this in a better way.

> On May 26, 2015, at 9:43 AM, Madhav Puri <notifications@github.com> wrote:
> 
> In netdcli/netdcli.go <https://github.com/contiv/netplugin/pull/79#discussion_r31053925>:
> 
> >  		}
> >  	} else if opts.pktTag != "" {
> >  		_, err = strconv.Atoi(opts.pktTag)
> >  		if err != nil {
> >  			log.Fatalf("Error convertinng tag %s to integer \n", opts.pktTag)
> > +			os.Exit(1)
> We may not need explicit os.Exit(1) after call to log.Fatalf() as I think Fatalf() will call exit.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/79/files#r31053925>.
>